dali/operators/decoder/image_decoder.cc

```diff
@@ -24,79 +24,110 @@ DALI_SCHEMA(ImageDecoderAttr)
-during the pipeline execution. One way to find the ideal value is to do a full run over the dataset with the
-argument ``memory_stats`` set to True and then copy the "biggest" allocation value printed in the statistics.)code",
+      R"code(Applies **only** to the ``mixed`` backend type.
+
+The padding for nvJPEG’s host memory allocations, in bytes. This parameter helps to prevent
+the reallocation in nvJPEG when a larger image is encountered, and the internal buffer needs
+to be reallocated to decode the image.
+
+If a value greater than 0 is provided, the operator preallocates two (because of double-buffering)
+host-pinned buffers of the requested size per thread. If selected correctly, no additional
+allocations will occur during the pipeline execution. One way to find the ideal value is to
+do a complete run over the dataset with the ``memory_stats`` argument set to True, and then copy
+the largest allocation value that is printed in the statistics.)code",
       8*1024*1024)  // based on ImageNet heuristics (8MB)
   .AddOptionalArg("affine",
-      R"code(**`mixed` backend only** If internal threads should be affined to CPU cores)code",
+      R"code(Applies **only** to them ``mixed`` backend type.
+
+If set to True, each thread in the internal thread pool will be tied to a specific CPU core.
+Otherwise, the threads can be reassigned to any CPU core by the operating system.)code",
       true)
   .AddOptionalArg("split_stages",
-      R"code(**`mixed` backend only** Split into separated CPU stage and GPU stage operators)code",
+      R"code(Applies **only** to the ``mixed`` backend type.
+
+If True, the operator will be split into two sub-stages: a CPU and GPU one.)code",
       false)
   .AddOptionalArg("use_chunk_allocator",
-      R"code(**Experimental, `mixed` backend only** Use chunk pinned memory allocator, allocating chunk of size
-`batch_size*prefetch_queue_depth` during the construction and suballocate them
-in runtime. Ignored when `split_stages` is false.)code",
+      R"code(**Experimental**, applies **only** to the ``mixed`` backend type.
+
+Uses the chunk pinned memory allocator and allocates a chunk of the
+``batch_size * prefetch_queue_depth`` size during the construction and suballocates
+them at runtime. When ``split_stages`` is false, this argument is ignored.)code",
       false)
   .AddOptionalArg("use_fast_idct",
-      R"code(Enables fast IDCT in CPU based decompressor when GPU implementation cannot handle given image.
-According to libjpeg-turbo documentation, decompression performance is improved by 4-14% with very little
-loss in quality.)code",
+      R"code(Enables fast IDCT in the libjpeg-turbo based CPU decoder, used when ``device`` is set
+to "cpu" or when the it is set to "mixed" but the particular image can not be handled by
+the GPU implementation.
+
+According to the libjpeg-turbo documentation, decompression performance is improved by up to 14%
+with a low reduction in quality.)code",
```

Comment:
```suggestion
with very little reduction in quality.)code",
```
That just sounds bad. It was meant to indicate that the loss of quality is in most cases negligible.

Reply:
Done